modules/js-and-css/embed-optimizer/hooks.php

```diff
@@ -63,8 +63,10 @@ function embed_optimizer_filter_oembed_html( string $html ): string {
 	if ( 1 === $script_count && ! $has_inline_script && $p->has_bookmark( 'script' ) ) {
 		add_action( 'wp_footer', 'embed_optimizer_lazy_load_scripts' );
 		if ( $p->seek( 'script' ) ) {
-			$p->set_attribute( 'data-lazy-embed-src', $p->get_attribute( 'src' ) );
-			$p->remove_attribute( 'src' );
+			if ( $p->get_attribute( 'type' ) ) {
+				$p->set_attribute( 'data-original-type', $p->get_attribute( 'type' ) );
+			}
+			$p->set_attribute( 'type', 'text/x.lazy-loaded-javascript' );
```

Comment:
Should this match the plugin name? maybe something like `text/embed-optimizer`?

Reply:
I tried to follow the rules outlined in https://en.wikipedia.org/wiki/Media_type

That said, what about `application/vnd.embed-optimizer.javascript`?

Reply:
Committed this in 39d44880